package com.class34FinallyBlock;

public class TestException {

    public static void main(String[] args) {
        TryCatchFinally.division(10, 2);
        TryCatchFinally.division(10, 0);
        System.out.println("----end of the code-----");

        TryCatchFinally.read("");
    }
}
